### DISCUSSION TITLE
Missing parts in the downloaded Twitch VOD

### DISCUSSION THREAD

**Discussion by xalyy** [streamlink/streamlink#5112] [Q&A] [answered]
Hello, 

I am using streamlink-5.1.1-1-py310-x86_64 and my issue is that the streams that I download live have missing parts in them. Usually around 30 second parts that are missing and it only occurs a couple of times throughout a several hour long VOD.
I am guessing it could be mid stream ads because if I download the VOD using the m3u8 link with ffmpeg only instead with streamlink, it doesnt have any interruptions, so unless I have missed a command or something I dont think the issue is on my side or the streamers side.

This is the command that im using: streamlink.exe --twitch-disable-ads --retry-streams=10 -o "E:/Backup/Twitch/examplestreamer_{time:%Y%}.mp4" https://www.twitch.tv/examplestreamer best

Thanks for the help.

**Comment by bastimeyer** [comment, marked as answer]
> `--twitch-disable-ads`
> Skip embedded advertisement segments at the beginning or during a stream. **Will cause these segments to be missing from the output.**

https://streamlink.github.io/cli.html#cmdoption-twitch-disable-ads
https://streamlink.github.io/cli/plugins/twitch.html#embedded-ads

**Reply by xalyy** [reply]
And is there no way to not get ads but have the missing parts?

**Reply by bastimeyer** [reply]
https://github.com/streamlink/streamlink/issues/4949
